app/controllers/admin/admin_statistic_controller.rb

```diff
@@ -0,0 +1,20 @@
+class Admin::AdminStatisticController < Admin::BaseAdminController
+  def index
+    return unless params[:year]
+    years = User.pluck(:created_at).map {|item| item.year}.uniq
+    @arrayDataPoints = Hash.new
```

Comment:
```suggestion
    @arrayDataPoints = {}
```
dùng như này cho ngắn